### PR TITLE
Update all patterns to use SVG icons

### PR DESCRIPTION
**Pull request by nikolaswise**

### CODE REVIEW

File: lib/js/helpers/classy.js [hunk 1]
```diff
@@ -7,26 +7,26 @@ import * as dom from './dom';
 
 // check if an element has a specific class
 export function has (domNode, className) {
-  var elementClass = ' ' + domNode.className + ' ';
-  return elementClass.indexOf(' ' + className + ' ') !== -1;
+  return new RegExp('(\\s|^)' + className + '(\\s|$)').test(domNode.getAttribute('class'));
```
Comment (paulcpederson) [inline lib/js/helpers/classy.js]: what is this?

File: lib/sass/calcite-web/components/_dropdown.scss [hunk 2]
```diff
@@ -41,27 +41,8 @@
   }
 
   @mixin dropdown-btn {
-    padding-right: 1.5rem;
     cursor: pointer;
     position: relative;
-    &:after {
```
Comment (paulcpederson) [inline lib/sass/calcite-web/components/_dropdown.scss]: so we have to add the svg code everywhere there is a dropdown?

File: lib/sass/calcite-web/patterns/_filter-dropdown.scss [hunk 3]
```diff
@@ -45,11 +45,29 @@
...
+@mixin filter-link-check() {
+  opacity: 0;
+  visibility: hidden;
+  margin-top: 6px;
+  float: left;
+  position: absolute;
+  left: 9px
+}
+@mixin filter-link-close() {
+  opacity: 0;
```
Comment (paulcpederson) [inline lib/sass/calcite-web/patterns/_filter-dropdown.scss]: so why float it *and* position it absolutely?

File: lib/js/helpers/classy.js (hunk 1 above)
Comment (nikolaswise) [inline lib/js/helpers/classy.js]: This is a regexp that checks if a class exists in the string returned from get attribute. Becuase SVG's dont have a `domNode.class` property that behaves the same as other actuall dom nodes. This lets the classy helpers work for HTML and SVG nodes.

File: lib/sass/calcite-web/components/_dropdown.scss (hunk 2 above)
Comment (nikolaswise) [inline lib/sass/calcite-web/components/_dropdown.scss]: Yes.

File: lib/sass/calcite-web/patterns/_filter-dropdown.scss (hunk 3 above)
Comment (nikolaswise) [inline lib/sass/calcite-web/patterns/_filter-dropdown.scss]: Super interesting question let me look.

Comment (nikolaswise) [inline lib/sass/calcite-web/patterns/_filter-dropdown.scss]: no reason, will delete float

**Comment by paulcpederson** [comment]
cool I'm going to merge this, but we should think about ways to reuse svgs on a page

**Pull request merged**